common/src/main/java/gov/cms/ab2d/common/model/CoverageSearch.java

```diff
@@ -0,0 +1,37 @@
+package gov.cms.ab2d.common.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+import static javax.persistence.EnumType.STRING;
+
+@Entity(name = "bene_coverage_search")
+@Getter
+@Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class CoverageSearch {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @EqualsAndHashCode.Include
+    private Integer id;
+
+    @ManyToOne
+    @JoinColumn(name = "contract_id")
+    private Contract contract;
+
+    @Column
+    @NotNull
+    private Integer month;
```

Comment:
If we are @NotNull, then why not use int instead of Integer?  int can not be null.

Reply:
Would not null annotation even apply to an integer?